units/provisioning_create_host.go

```diff
@@ -122,23 +122,31 @@ func (j *createHostJob) createHost(ctx context.Context) error {
 		return errors.Wrapf(errIgnorableCreateHost, "problem getting cloud provider for host '%s' [%s]", j.host.Id, err.Error())
 	}
 
-	// On the first attempt, remove the intent document so no other create host job tries to create this intent.
+	if err := j.host.SetStatus(evergreen.HostSpawning, evergreen.User, ""); err != nil {
+		return errors.Wrapf(err, "problem setting host %s status to spawning", j.host.Id)
+	}
+
+	defer j.tryRequeue(ctx)
```

Comment:
Should this go above line 125? That way if there was an error setting the status, we try this again

Reply:
Fixed.